Parsers/ASimWebSession/Parsers/vimWebSessionzScalerZIA.yaml

```diff
@@ -149,7 +150,12 @@ ParserQuery: |
     EventSeverity = case (ThreatRiskLevel > 90, "High", ThreatRiskLevel > 60, "Medium", ThreatRiskLevel > 10, "Low",  "Informational"),
     DstAppType = "SaaS application",
     DstDomainType = iff (DstHostnameNotAddr, "FQDN", ""),
-    SrcUsernameType = "UPN"
+    SrcUsernameType = "UPN",
+    ASimMatchingIpAddr = case(array_length(ipaddr_has_any_prefix) == 0 ,"",
```

Comment:
again, do this is the pre-filter to reduce calls to has_any_ipv4

Reply:
This is not a filter. The events are already filtered at line 85. BUT: since these are already filtered I CAN improve this a bit. 
I could: extend SrcMatch = has_any_ipv4_prefix(SrcIpAddr,ipaddr_has_any_prefix), DstMatch = has_any_ipv4_prefix(DstIpAddr,ipaddr_has_any_prefix)
and then use this column instead of calling has_any_ipv4.

Reply:
Exactly, use your extend message in line 85 (breaking the where clause), and use the values in the extend in 154.